Bot Protection docs (#8893)
auth0/docs: articles/anomaly-detection/concepts/credential-stuffing.md

---
description: Understand how Auth0 detects anomalies to stop malicious attempts to access your application, alert you and your users of suspicious activity, and block further login attempts. 
topics:
    - security
    - anomaly-detection
    - credential-stuffing
contentType: concept
useCase: anomaly-detection
---

> # Automated Attack and Credential Stuffing Protection

Reviewer: Is this really "Automated Attack" *and* "Credential Stuffing" protections?  Again, use of Automated Attack is unclear - is it the same?  Is Credstuff a type of automated attack?  If so, why "and" ?  If not, why are we including Automated Attack here?

> Credential stuffing attacks (or *list validation attacks*) occur when bad actors automate the process of trying username and password combinations (usually stolen from another site) for many accounts in a short period of time.  According to recent statistics, as many as 71% of accounts use the same password across multiple sites so a credential stuffing attack has the potential to successfully log into your system.  

Reviewer: ```suggestion
Credential stuffing attacks (or *list validation attacks*) occur when bad actors automate the process of trying username and password combinations (usually stolen from another site) for many accounts in a short period of time.  According to recent statistics, as many as 71% of accounts use the same password across multiple sites so a credential stuffing attack represents a legitimate threat to your system.  
```

> Use **Automated Attack Protection** to provide a standard level of protection against credential stuffing attacks that does not add any friction to legitimate users. This protection is enabled by default for all connections.  

Reviewer: I'd be careful with "does not add any friction to legitimate users", as we WILL have false positives and do things like throw captcha for a small number of legit users.

```suggestion
Use **Automated Attack Protection** to provide a standard level of protection against credential stuffing attacks with minimal friction for legitimate users. This protection is enabled by default for all connections.  
```

::: panel Credential Stuffing Attacks: What Are They and How to Combat Them
Download this free [whitepaper](https://auth0.com/resources/whitepapers/credential-stuffing-attacks) to learn how Auth0 can help you combat credential stuffing attacks.
:::

## How it works

> Auth0 uses a large amount of data to identify patterns that signal that a credential stuffing attack is taking place. Auth0 uses data algorithms to determine when bursts of traffic are likely to be from a bot or script. Users attempting to sign in from IPs which are determined to have a high likelihood of being a credential stuffing attack will see a CAPTCHA step. The algorithms are designed so that this only happens for bad traffic; the objective is to not show any friction to legitimate users.

Reviewer: This is a GREAT description of *what* credstuff does without exposing the *how* to our customers.  I'd recommend using text similar to this on the attack-protection-triggers-actions page (see comment there).
